k8s/plugin/src/main.rs

```diff
@@ -166,6 +167,7 @@ async fn execute(cli_args: CliArgs) {
                         std::process::exit(1);
                     });
                 println!("Completed collection of dump !!");
+                Ok(())
```

Comment:
Should remove exit above?

Reply:
We could remove the exit code. But different arms return different error types (plugin errors and upgrade errors). Should I use custom error enum?